flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
Any reason this is x86_64-only?

Reply:
The build on ARM64 is currently failing, as mentioned in [this comment](https://github.com/flathub/flathub/pull/3335#issuecomment-1191758816). There is already a fix upstream, but I do not want to add it as a patch because it was not tested yet (and I currently do not have any ARM hardware here at the moment). I will wait for the new SDRangel release with the fix hopefully included.